chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+
+3. Digital Image Formation and Representation
+The core of digital image formation is the function f(x, y), which is determined by the illumination source i(x, y), and the reflectance r(x, y) from the scene. In transmission-based imaging, such as X-rays, transmissivity takes the place of reflectivity. The digital representation of an image is essentially a matrix or array of numerical values, each corresponding to a pixel. The process of transforming continuous image data into a digital format is twofold: sampling, which digitizes the coordinate values, and quantization, which converts amplitude values into discrete quantities. The resolution and quality of a digital image are significantly dependent on the number of samples and discrete intensity levels used. The dynamic range of the imaging system, which is the ratio of the maximum measurable intensity to the minimum detectable intensity, also plays a crucial role in the appearance and contrast of the image.
+
+4. Understanding Resolution in Digital Imaging
+Spatial resolution refers to the smallest distinguishable detail in an image and is often measured in line pairs per unit distance or pixels per unit distance. The meaningfulness of spatial resolution is context-dependent, varying according to the spatial units used. For example, a 20-megapixel camera typically offers higher detail resolution than an 8-megapixel camera. Intensity resolution relates to the smallest detectable change in intensity level and is often limited by the hardware's capabilities. It's quantized in binary increments, such as 8 bits or 256 levels. The perception of these intensity changes is influenced by various factors, including noise, saturation, and the capabilities of human vision.
+
+5. Various Operations in Digital Image Processing

```

Comment:
5, 6, 7 feel really out of place here. 
It fits more on the pre-processing chapter.